src/renderer/features/operations/OperationsConfirm/Referendum/RemoveVote/model/confirm-model.ts

```diff
@@ -0,0 +1,34 @@
+import { type ApiPromise } from '@polkadot/api';
+import { createEvent } from 'effector';
+
+import { type AccountVote, type Asset, type ReferendumId, type TrackId } from '@/shared/core';
+import { walletModel } from '@/entities/wallet';
+import { submitModel } from '@/features/operations/OperationSubmit';
+import { type ConfirmInfo, createTransactionConfirmStore } from '@/features/operations/OperationsConfirm';
+
+export type RemoveVoteConfirm = ConfirmInfo & {
+  api: ApiPromise;
+  asset: Asset;
+  referendumId: ReferendumId;
+  trackId: TrackId;
+  vote: AccountVote;
+};
+
+const sign = createEvent();
+
+const confirmStore = createTransactionConfirmStore<RemoveVoteConfirm>({
+  $wallets: walletModel.$wallets,
+});
+
+export const confirmModel = {
+  $confirmMap: confirmStore.$confirmMap,
+  events: {
+    sign,
+    addConfirms: confirmStore.addConfirms,
+    replaceWithConfirm: confirmStore.replaceWithConfirm,
+    fillConfirm: confirmStore.fillConfirm,
+
```

Comment:
don't forget to use `resetConfirm` after you finish with the flow. It'll prevent from possibility of incorrect data in submit modal later